### PR TITLE
Add a flag for formatting bzl files

### PR DESCRIPTION
**Pull request by vladmos**

### CODE REVIEW

File: buildifier/buildifier.go [hunk 1]
```diff
@@ -46,9 +46,13 @@ var (
 	tablesPath    = flag.String("tables", "", "path to JSON file with custom table definitions which will replace the built-in tables")
 	addTablesPath = flag.String("add_tables", "", "path to JSON file with custom table definitions which will be merged with the built-in tables")
 	version       = flag.Bool("version", false, "Print the version of buildifier")
+
 	// Debug flags passed through to rewrite.go
 	allowSort = stringList("allowsort", "additional sort contexts to treat as safe")
 	disable   = stringList("buildifier_disable", "list of buildifier rewrites to disable")
+
+	// Experimental flags
+	formatBzlFiles = flag.Bool("formatbzl", false, "format bzl-specific blocks")
```
[the remark this answers is not in the document]
Comment (vladmos) [reply buildifier/buildifier.go]: formatBzl? Other flags are camelcased here.

[the remark this answers is not in the document]
Comment (vladmos) [reply buildifier/buildifier.go]: Or, do you mean the flag name, not the variable? I'll rename it.

File: buildifier/buildifier.go [hunk 2]
```diff
@@ -46,9 +46,13 @@ var (
 	tablesPath    = flag.String("tables", "", "path to JSON file with custom table definitions which will replace the built-in tables")
 	addTablesPath = flag.String("add_tables", "", "path to JSON file with custom table definitions which will be merged with the built-in tables")
 	version       = flag.Bool("version", false, "Print the version of buildifier")
+
 	// Debug flags passed through to rewrite.go
 	allowSort = stringList("allowsort", "additional sort contexts to treat as safe")
 	disable   = stringList("buildifier_disable", "list of buildifier rewrites to disable")
+
+	// Experimental flags
+	formatBzlFiles = flag.Bool("format_bzl", false, "format bzl-specific blocks")
```
Comment (laurentlb) [inline buildifier/buildifier.go]: Add "Experimental" or any kind of warning.

**Comment by laurentlb** [comment]
lgtm

**Comment by pmbethe09** [comment]
LGTM (yes I had mean the program flag name, not variable -- and you figured that out)

**Pull request merged**